source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -461,3 +463,86 @@ def include_poi_access_points(request, pt_object, mode):
         ]
         and pt_object.poi.children
     )
+
+
+def get_impact_uris_for_poi(response, poi):
+    impact_uris = []
```

Comment:
```suggestion
    impact_uris = set()
```

Reply:
be careful... in case of /places_nearby, the complexity will be O(N*N)... which means, for each place in `places_nearby`, you have to scan the whole `response.impacts`... could lead to some important lantence

Reply:
use a map in this case

Reply:
For the time being we should have only some impacts on pois.